packages/tools/generator-widget/generators/app/templates/packages/package_web.json.ejs

```diff
@@ -23,8 +23,9 @@
     "lint:fix": "pluggable-widgets-tools lint:fix",<% if (hasUnitTests) { %>
     "test": "pluggable-widgets-tools test:unit:web --no-cache --ci<% if (hasE2eTests) { %> && npm run test:e2e<% } %>",
     "test:unit": "pluggable-widgets-tools test:unit:web --coverage",<% } %><% if (hasE2eTests) { %>
-    "test:e2e": "pluggable-widgets-tools test:e2e --url \"https://mywidget.mxapps.io\"",
-    "test:e2e:dev": "pluggable-widgets-tools test:e2e:web:dev",<% } %>
+    "pretest:e2e": "npm run release && node ../../../scripts/test/updateAtlas.js --latest-atlas",
+    "test:e2e": "pluggable-widgets-tools test:e2e:web:cypress",
+    "test:e2e:local": "npx cypress open --browser chrome --e2e --config-file ../configs/e2e/cypress/cypress.config.js",<% } %>
```

Comment:
Hm, if we use `cypress` directly, and not via `pluggable-widgets-tools`, this probably means we have to add cypress as dependencies to the `package.json` of the widget itself. 🤔  

Maybe it is better to actually have a command in `pluggable-widgets-tools` that does run cypress locally. Like `test:e2e:web:cypress:local`?

Reply:
Yeah, I can do that. Just need to spread that to all widgets as well.

Reply:
Done!